components/auth/AuthProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import { loadAccessToken } from '@/components/auth/accessToken';
+import { FC, ReactNode, useRef } from 'react';
+
+export const AuthProvider: FC<{ children: ReactNode }> = (props) => {
+  useClientOnce(() => {
+    loadAccessToken();
+  });
```

Comment:
요거 `useEffect` 쓰는거랑 달라? 이거 실행하는 법을 모르겠어서 코드만 보고 얘기했엉
```suggestion
  useEffect(() => {
    loadAccessToken();
  }, []);
```

Reply:
useEffect 는 자식 컴포넌트 -> 부모 컴포넌트 순으로 호출되는데, 여기서는 이 동작이 문제가 될 수 있어! 자식 컴포넌트보다 먼저 로딩되는것을 보장하기 위해 해당 로직을 넣었어